backend/core/socketio_config.py

```diff
@@ -319,6 +322,39 @@ async def resend_messages(session: Session, user_id=int, user_sid=str):
     return last_chats
 
 
+def get_chat_history_for_assistant(
+    session: Session, chat_session_id: int, body: str
+):
+    last_chats = session.exec(
+        select(Chat)
+        .where(
+            and_(
+                Chat.chat_session_id == chat_session_id,
+                Chat.sender_role.in_(
+                    [
+                        Sender_Role_Enum.USER,
+                        Sender_Role_Enum.CLIENT,
+                        Sender_Role_Enum.ASSISTANT,
+                    ]
+                ),
+            )
+        )
+        .order_by(Chat.created_at.desc())
+        .offset(1)  # skip the latest message
```

Comment:
what happen if no chat at all?

Reply:
Hi mas @dedenbangkit , 
I pushed a test case to cover this scenario.